README.md

```diff
@@ -3,8 +3,54 @@
 This a pure Scala implementation of [Babelfish](https://doc.bblf.sh/) client.
 It uses [ScalaPB](https://scalapb.github.io/grpc.html) for Protobuf/gRPC code generation.
 
+### Installation
```

Comment:
It'd be nice to describe how to install it through a package manager too (something like https://github.com/src-d/enry/tree/master/java#usage)